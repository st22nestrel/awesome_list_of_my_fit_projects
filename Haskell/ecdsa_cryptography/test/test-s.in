Curve {
    p: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEFFFFFC2F
    a: 0
    b: 7
    g: Point {
    x: 0x79BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F81798
    y: 0x483ADA7726A3C4655DA4FBFC0E1108A8FD17B448A68554199C47D08FFB10D4B8
    }
    n: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141
    h: 1
}
Key {
    d: 0xc9dcda39c4d7ab9d854484dbed2963da9c0cf3c6e9333528b4422ef00dd0b28e
    Q: 0x040E411E56210F20BF5172CBEFAB02135421B1EB58F6918D28C1B848BE5EEE42E4C9ACA4E3F1B7360BFB0DBF735438C90989C7D980ACD42AF7A05EF9BE044954D2
}
Hash: 0x94996fead5b722c3bd07360c459927976e804f869626f4897def03fa56b009e3