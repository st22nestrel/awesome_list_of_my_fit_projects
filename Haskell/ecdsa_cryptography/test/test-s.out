Signature {
	r: 0x0701719bc8ead104f45f694dd57692b78196378c8fb67e473b35b65b300a2be4
	s: 0x5cb4b5e9eaa782dd29fb22bd63051038964c14278d0388436ab07fcf1da69db1bb1320b1b537c6009e0467f12632b93575eaaba371741146660773da566af090b61ea4d90e6837223490599006d4bc8e7cc6564e7ed967432961fe59b42aed
}
